recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "pyobjc-framework-FSEvents" %}
-{% set version = "10.2" %}
+{% set version = "10.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 3a75f38bb1d5d2cf6a0d3e92801b3510f32e96cf6443d81b9dd92a84d72eff0a
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|lower }}-{{ version }}.tar.gz
```

Comment:
FYI this is a relevant change -- the previous releases had `pyobjc-framework-FSEvents` in the sdist name but 10.3 and 10.3.1 sdists have `pyobjc-framework-fsevents`. All wheels have the capitalized version, and other packages like `pyobjc-framework-coreservices` kept caps in the sdist name, so it's probably a bug that the sdist is not capitalized.

Reply:
... argh, and underscores

Reply:
That probably explains why the bot didn't pick up on the new release